question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -375,8 +379,23 @@ def string_from_2darray(A, language='python', presentation_type='f', digits=2):
             return numpy_to_matlab_sf(A, ndigits=digits)
         else:
             return numpy_to_matlab(A, ndigits=digits, wtype=presentation_type)
+    elif language == 'mathematica':
+        if presentation_type == 'sigfig':
```

Comment:
How do we trigger `sigfig` presentation? Is there an option for this? I don't think it's documented.

Reply:
Also, the docs say that `digits` specifies "Number of digits to display after the decimal", which isn't true if we are doing `sigfig`.

Reply:
I have no idea how this works and actually noticed with my example question. The default "digits" for the element is `3`, when I set it to `2`, i think it displays the same number of digits after the decimal.

Reply:
Adding an option to switch between sigfig digits and digits-after-decimal would be really nice. I think we have examples in other elements of doing this, and if possible we should copy the attribute names and code (I think this will be fairly easy).